packages/core/parcel/package.json

```diff
@@ -14,7 +14,7 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "lib/bin.js",
```

Comment:
```suggestion
  "bin": "src/bin.js",
```

Reply:
Oops, yes, good catch - I accidentally committed this!